Sphinxsetup.bat

```diff
@@ -12,7 +12,6 @@ pip install --upgrade git+https://github.com/ArduPilot/sphinx_rtd_theme.git
 
 rem and a youtube plugin:
 pip install --upgrade git+https://github.com/ArduPilot/sphinxcontrib-youtube.git
-pip install --upgrade git+https://github.com/ArduPilot/sphinxcontrib.vimeo.git
```

Comment:
why remove the install of the vimeo plugin?

Reply:
It doesn't support the latexpdf builder. The sphinxcontrib.youtube extension does support Vimeo and the latexpdf builder. It's one of the steps towards getting a PDF version of the docs.

See also #4195 where it was removed. The version checking added in that PR can be removed now.